applications/nrf5340_audio/src/bluetooth/ble_acl_common.c

```diff
@@ -213,3 +230,163 @@ int ble_acl_common_init(void)
 
 	return 0;
 }
+
+static enum ble_hci_vs_tx_power selection_to_enum_val(int selection)
+{
+	switch (selection) {
+	case 0:
+		return BLE_HCI_VSC_TX_PWR_0dBm;
+	case 1:
+		return BLE_HCI_VSC_TX_PWR_Neg1dBm;
+	case 2:
+		return BLE_HCI_VSC_TX_PWR_Neg2dBm;
+	case 3:
+		return BLE_HCI_VSC_TX_PWR_Neg3dBm;
+	case 4:
+		return BLE_HCI_VSC_TX_PWR_Neg4dBm;
+	case 5:
+		return BLE_HCI_VSC_TX_PWR_Neg5dBm;
+	case 6:
+		return BLE_HCI_VSC_TX_PWR_Neg6dBm;
+	case 7:
+		return BLE_HCI_VSC_TX_PWR_Neg7dBm;
+	case 8:
+		return BLE_HCI_VSC_TX_PWR_Neg8dBm;
+	case 9:
+		return BLE_HCI_VSC_TX_PWR_Neg12dBm;
+	case 10:
+		return BLE_HCI_VSC_TX_PWR_Neg16dBm;
+	case 11:
+		return BLE_HCI_VSC_TX_PWR_Neg20dBm;
+	case 12:
+		return BLE_HCI_VSC_TX_PWR_Neg40dBm;
+	default:
+		return BLE_HCI_VSC_TX_PWR_INVALID;
+	}
+}
+
+static void shell_print_selection(const struct shell *shell)
+{
+	shell_print(shell, "Possible settings:");
+	shell_print(shell, "\t0: 0dBm");
+	shell_print(shell, "\t1: -1dBm");
+	shell_print(shell, "\t2: -2dBm");
+	shell_print(shell, "\t3: -3dBm");
+	shell_print(shell, "\t4: -4dBm");
+	shell_print(shell, "\t5: -5dBm");
+	shell_print(shell, "\t6: -6dBm");
+	shell_print(shell, "\t7: -7dBm");
+	shell_print(shell, "\t8: -8dBm");
+	shell_print(shell, "\t9: -12dBm");
+	shell_print(shell, "\t10: -16dBm");
+	shell_print(shell, "\t11: -20dBm");
+	shell_print(shell, "\t12: -40dBm");
+}
+
+static int cmd_ble_tx_pwr_conn(const struct shell *shell, size_t argc, char **argv)
+{
+	int ret;
+	struct bt_conn *conn;
+	uint8_t num_conn = 0;
+	uint16_t peer_conn_handle;
+
+	if (argc != 2) {
+		shell_error(shell, "Wrong number of arguments provided");
+		return -EINVAL;
+	}
+
+	if (strcmp(argv[1], "help") == 0) {
+		shell_print_selection(shell);
+		return 0;
+
```

Comment:
Remove newline

Reply:
Disagree, as long as there is an else if() below, a newline after a return helps readability